pypeit/core/telluric.py

```diff
@@ -135,11 +134,69 @@ def pca_eval(theta,pca_dict):
 #        theta_pca[2:npca+1]``
 #
 #    """
-#    gaussian_mixture_model = pca_dict['prior']
+#    gaussian_mixture_model = qso_pca_dict['prior']
 #    A = theta[2:]
 #    return gaussian_mixture_model.score_samples(A.reshape(1,-1))
 
 
+def read_telluric_pca(filename, wave_min=None, wave_max=None, pad_frac=0.10):
+    """
+    Reads in the telluric PCA components from a file.
+
+    Optionally, this method also trims wavelength to be in within ``wave_min``
+    and ``wave_max`` and pads the data (see ``pad_frac``).
+
+    .. todo::
+        List and describe the contents of the dictionary in the return
+        description.
+
+    Args:
+        filename (:obj:`str`):
+            Telluric PCA filename
+        wave_min (:obj:`float`):
+            Minimum wavelength at which the grid is desired
+        wave_max (:obj:`float`):
+            Maximum wavelength at which the grid is desired.
+        pad_frac (:obj:`float`):
+            Percentage padding to be added to the grid boundaries if
+            ``wave_min`` or ``wave_max`` are input; ignored otherwise. The
+            resulting grid will extend from ``(1.0 - pad_frac)*wave_min`` to
+            ``(1.0 + pad_frac)*wave_max``.
+
+    Returns:
+        :obj:`dict`: Dictionary containing the telluric PCA components.
```

Comment:
Please document the elements of the dict.

Reply:
Done, and added similar documentation to `read_telluric_grid`